clang/test/SemaCXX/init-priority-attr.cpp

```diff
@@ -24,32 +24,35 @@ extern Two goo;
 extern Two coo[];
 extern Two koo[];
 
+// unknown-system-no-diagnostics
```

Comment:
Can you also add a new `RUN` line like:
```
// RUN: %clang_cc1 -triple=x86_64-unknown-unknown -fsyntax-only -Wno-error=init-priority-reserved -verify %s
```
to verify that we get warnings rather than errors when we downgrade the diagnostic level?